content/syllabuses/data-sci.md

```diff
@@ -32,11 +32,13 @@ _db_id: 2
 - {{% contentlink path="projects/data-science-specific/cross-validation-and-simple-linear-regression/" flavour="python" %}}
 - {{% contentlink path="topics/intro-to-data-ethics" %}}
 - {{% contentlink path="topics/data-ethics-and-privacy" %}}
+- {{% contentlink path="topics/solo-learn/python/9-regular-expressions" %}}
 - {{% contentlink path="projects/tdd/string-calculator" flavour="python" %}}
```

Comment:
Added regex before string calc, but this syllabus does look a bit weird and ot of order to me in some areas.

i.e
1. oop for  python is on line 22 but animals is all the way down on line 38.
2. string calc comes before oop? not sure if this was intentional

Reply:
Those can be reordered. But it's probably not a big deal because the prerequisites come out in a way that supports the order laid out